lib/cpp/src/thrift/concurrency/Mutex.cpp

```diff
@@ -37,11 +37,10 @@ namespace concurrency {
 
 #ifndef THRIFT_NO_CONTENTION_PROFILING
 
+static sig_atomic_t mutexProfilingCounter = 0;
 static sig_atomic_t mutexProfilingSampleRate = 0;
```

Comment:
According to the comment in the implementation file, a race on this variable is acceptable, so I will change the type as you suggested and re-submit.